assume/world.py

```diff
@@ -129,6 +129,7 @@ def __init__(
             "power_plant": PowerPlant,
             "demand": Demand,
             "storage": Storage,
+            "steel_plant": SteelPlant,
```

Comment:
I don't think the real specialty of this model is steel?
In the long term, I would like to have a more generic solution, for example, something that also works for a pulp machine or other energy-intensive industries.
Having "Demand", "PowerPlant", "Storage" and then "Steelplant" seems a little weird.
But the steel plant is more of a general modeling approach to a multi step industrial process right?

Reply:
The "SteelPlant" in this model focuses on integrating technologies specific to steel production. While I aim to create a more generic solution for various energy-intensive industries in the long term, currently, the model is tailored to the steel industry. When in the next step I develop similar models for other industries like cement will help identify commonalities for a unified approach in the future.